docs/accessories/developer/api_fundamentals.md

```diff
@@ -0,0 +1,44 @@
+
+!!! warning "Default Rendering Behavior"
+    By default as discussed within the Rendering API section of the docs, all Accessories will use `DefaultAccessoryRenderer` unless a custom `AccessoryRenderer` is registered or no renderer is registered using `AccessoriesRendererRegistry#registerNoRenderer`.
+
+### Creating a custom Slot Predicate
+
+In cases where a Tag may not be the best solution or using Data Components on your ItemStack may not be suitable, one can create a custom `SlotBasedPredicate` where you can have more general control of allowing `Accessory` entries to be equipped to a given slot. 
+
+This is useful for dynamic accessory types that may have multiple iterations that would make it difficult to use a Item Tag without possible issues.
+
+!!! info "Entity Based Predicate"
+    With the release of 1.2.0, `EntityBasedPredicate` exists as a way to have entity context when evaluating if a given Accessory should be equipped within a given slot. It is recommend that this is used only if needed as it may be difficult to evaluate for things like crafting where the target entity may differ.
+
+### Accessing The Capability
+
+Within Accessories there is the `AccessoriesCapability` which is used to access the slot containers, adjust slot attributes, or query the inventory for various possible item stacks. 
+
+Any interaction involving the entity would be found here if they have a `AccessoriesCapability` as the entity needs to have slots bound to such to be able to get the capability. This means that your logic should revolve around the idea that not every entity can be capable of being interacted with unless they have slots bound to them.
+
+### Adjusting Accessory Equipping
+
+A considerable difference between other Accessory API's is that `canEquip` is checked on loading Accessory data and revalidate on reload requiring that you use `isAnotherEquipped` found on the given entities `AccessoriesCapability` to prevent duplicate Accessory items from being equipped. Such requires the current `ItemStack` and `SlotReference` of the current location to be ignored when checking for another equipped accessory.
+
+### Dealing with the Container
+
+For each slot there will be an `AccessoriesContainer` instance if the given Entity has the slot bound to such. That means that it has the possibility to not exist for every entity meaning you should confirm such to exist in all cases.
+
+This object allows for getting information like the current equipped accessories, render toggles for given accessories and managing the slot attributes stored. It mainly not recommend to directly use the container to look for Accessories but instead use the methods on `AccessoriesCapability` instead.
```

Comment:
```suggestion
This object allows for getting information like the current equipped accessories, render toggles for given accessories and managing the slot attributes stored. It is not recommended to directly use the container to look for Accessories. Use the methods on the `AccessoriesCapability` instead.
```
Is there a reason these methods which should not be used are accessible in the API?

Reply:
It's mainly down to the fact that it's undecided whether access to such is a good idea directly from the `AccessoryContainer`, but it may be useful for people implementing a transfer API to access the direct Inventory/Container instances for the Accessory Container.